app-admin/vuls/vuls-0.9.0-r1.ebuild

```diff
@@ -262,16 +261,13 @@ RDEPEND="
 	dev-go/goval-dictionary[policykit=]
 	dev-go/gost[policykit=]
 	dev-go/go-exploitdb[policykit=]
-	policykit? ( sys-auth/polkit )
+	policykit? (
+		acct-group/vuls
+		acct-user/vuls
```

Comment:
maybe it also needs changing for ebuilds of these packages
```
dev-go/go-cve-dictionary
dev-go/gost
dev-go/goval-dictionary
dev-go/go-exploitdb
dev-go/go-exploitdb
www-apps/vulsrepo
```

Reply:
Thanks, also changed those in 52a80a6 now